src/swarm-engine/Swarm/Game/Step.hs

```diff
@@ -120,14 +127,17 @@ gameTick = measureCpuTimeInSec runTick >>= updateMetrics
   runTick :: m Bool
   runTick = do
     time <- use $ temporal . ticks
-    zoomRobots $ wakeUpRobotsDoneSleeping time
-    ticked <- runActiveRobots
-    updateBaseReplState
-    -- Possibly update the view center.
-    modify (robotInfo %~ recalcViewCenter)
-    -- On new tick see if the winning condition for the current objective is met
-    when ticked hypotheticalWinCheck'
-    return ticked
+    Effect.localData ["tick" Effect..= time] $ do
+      Effect.logTrace_ "Running tick"
```

Comment:
@byorgey I realized I have not set a goal in #2526, what should we start logging for the issue to be _done_.

Do you have any suggestions? I am thinking:
* loading each part of state
* scenario start
* scenario end

Maybe also something in TUI to see how it would work in `EventM` monad.

Reply:
That sounds like a good start.